scripts/spans_oom_reproduce.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
Does it make sense to change this to `set -euxo pipefail`? The `-x` option is supposed to print all executed commands on the terminal. Since this is a debug script, I'd assume a developer to run this manually. If the commands in the script are printed to the terminal, it will help in debugging.